ga4gh/cli.py

```diff
@@ -1682,6 +1692,33 @@ def func():
             self._updateRepo(self._repo.removeDataset, dataset)
         self._confirmDelete("Dataset", dataset.getLocalId(), func)
 
+    def addFeatureSet(self):
+        """
+        Adds a new feature set into this repo
+        """
+        self._openRepo()
+        dataset = self._getDatasetFromArgs()
+        name = getNameFromPath(self._args.filePath)
+        featureSet = sequenceAnnotations.Gff3DbFeatureSet(
+            dataset, name)
+        referenceSet = self._repo.getReferenceSetByName(
+            self._args.referenceSetName)
```

Comment:
What happens when ``referenceSetName`` is None? Have we any hope of guessing the correct reference set from a given FeatureSet? We should output an explicit error message here anyway.